tasks/libs/package/size.py

```diff
@@ -0,0 +1,121 @@
+import os
+import tempfile
+from datetime import datetime
+
+from tasks.libs.common.color import color_message
+
+DEBIAN_OS = "debian"
+CENTOS_OS = "centos"
+SUSE_OS = "suse"
+
+SCANNED_BINARIES = {
+    "agent": {
+        "agent": "opt/datadog-agent/bin/agent/agent",
+        "process-agent": "opt/datadog-agent/embedded/bin/process-agent",
+        "trace-agent": "opt/datadog-agent/embedded/bin/trace-agent",
+        "system-probe": "opt/datadog-agent/embedded/bin/system-probe",
+        "security-agent": "opt/datadog-agent/embedded/bin/security-agent",
+    },
+    "iot-agent": {
+        "agent": "opt/datadog-agent/bin/agent/agent",
+    },
+    "dogstatsd": {
+        "dogstatsd": "opt/datadog-dogstatsd/bin/dogstatsd",
+    },
+    "heroku-agent": {
+        "agent": "opt/datadog-agent/bin/agent/agent",
+        "process-agent": "opt/datadog-agent/embedded/bin/process-agent",
+        "trace-agent": "opt/datadog-agent/embedded/bin/trace-agent",
+    },
+}
+
+
+def extract_deb_package(ctx, package_path, extract_dir):
+    ctx.run(f"dpkg -x {package_path} {extract_dir} > /dev/null")
+
+
+def extract_rpm_package(ctx, package_path, extract_dir):
+    with ctx.cd(extract_dir):
+        ctx.run(f"rpm2cpio {package_path} | cpio -idm > /dev/null")
+
+
+def extract_package(ctx, package_os, package_path, extract_dir):
+    if package_os == DEBIAN_OS:
+        return extract_deb_package(ctx, package_path, extract_dir)
+    elif package_os == CENTOS_OS or package_os == SUSE_OS:
```

Comment:
NIT: I think this syntax is easier to read
```suggestion
    elif package_os in (CENTOS_OS, SUSE_OS):
```